packages/amplify-graphql-api-construct-tests/src/__tests__/default-ddb-canary.test.ts

```diff
@@ -0,0 +1,33 @@
+import * as path from 'path';
+import { createNewProjectDir, deleteProjectDir } from 'amplify-category-api-e2e-core';
+import { cdkDestroy } from '../commands';
+import { setupBackend, testGraphQLOperations } from '../canary-tests-common';
+
+jest.setTimeout(1000 * 60 * 60 /* 1 hour */);
+
+describe('Canary using default DynamoDB model datasource strategy', () => {
+  let projRoot: string;
+  let projFolderName: string;
+
+  beforeEach(async () => {
+    projFolderName = 'defaultddbcanary';
+    projRoot = await createNewProjectDir(projFolderName);
+  });
+
+  afterEach(async () => {
+    try {
+      await cdkDestroy(projRoot, '--all');
+    } catch (_) {
+      /* No-op */
+    }
+
+    deleteProjectDir(projRoot);
+  });
+
+  test('Able to deploy simple schema', async () => {
+    const templatePath = path.resolve(path.join(__dirname, 'backends', 'base-cdk'));
```

Comment:
What is the different between this `base-cdk` canary and those running in the `e2e` workflows, which also has the `base-cdk` tests in all regions?

Reply:
We are re-using the same backend we have for `base-cdk` E2E in this `default-ddb-canary` here. This canary is intended to be run on `release` branch while this E2E runs on `main`, so there is some value to keep the regionalized test for now. Also the GraphQL operations we cover in `base-cdk` E2E is only a subset of the ones we cover in the canary.